src/pk.c

```diff
@@ -51,6 +51,487 @@ static int pem_write_bio_pubkey(WOLFSSL_BIO* bio, WOLFSSL_EVP_PKEY* key);
 
 #if defined(OPENSSL_EXTRA)
 
+#if (!defined(NO_DH) && (defined(HAVE_LIGHTY) || defined(HAVE_STUNNEL) || \
+     defined(WOLFSSL_MYSQL_COMPATIBLE) || defined(OPENSSL_EXTRA))) || \
+    (defined(OPENSSL_EXTRA) && !defined(WOLFCRYPT_ONLY))
+#ifndef NO_BIO
+
+#ifdef WOLFSSL_NO_FSEEK
+/* Amount of memory to allocate/add. */
+#define READ_BIO_FILE_CHUNK     128
+
+/* Read a file in chunks.
+ *
+ * Allocates a chunk and reads into it until it is full.
+ *
+ * @param [in, out] bio  BIO object to read with.
+ * @parma [out]     pem  Read data in a new buffer.
+ * @return  -ve on error.
```

Comment:
-ve -> negative. Please change elsewhere too. I don't think many people are familiar with this abbreviation.

Reply:
Global search and replace.